### ISSUE TITLE
このドライバはCentOS7のみですか？

### ISSUE THREAD

**Issue by izenizen** [project-pp/px_drv#1]
このドライバを使えれば、じゃじゃ馬wx-q3peが使えるのですが、CentOS7のみでしょうか？
Ubuntuでも動かせるようにサポートしていただけませんか？

**Comment by project-pp** [comment]
@izenizen 
Please tell me if I don't understand what you mean.
Have you seen `modinfo pxq3pe_dtv.ko`? 

```
$ modinfo pxq3pe_dtv.ko
filename:       /lib/modules/2.6.32-573.el6.x86_64/kernel/drivers/video/pxq3pe_dtv.ko
license:        GPL
author:
description:    PX-Q3PE driver module
srcversion:     A6E2A26B94BF482246AB236
alias:          pci:v0000188Bd00005220sv00000B06sd00000002bc*sc*i*
depends:
vermagic:       2.6.32-279.el6.x86_64 SMP mod_unload modversions
parm:           irq_debug:enable debug messages [IRQ handler] (int)
parm:           vid_limit:capture memory limit in megabytes (int)
```

According to the infomation, this module is compatible with PX-Q3PE on kernel 2.6.32. Now CentOS 7 is based on 3.x, and recent Ubuntu is also based on 3.x or 4.x kernel. Most of kernel update may not have backward compatibility, sometimes change interface or remove API. Generally we need to build kernel module against that a specific kernel version. But I have no source code, there is no way to implove compatibility.

__TL;DR: This driver is only compatible with kernel 2.6.32. I was no able to fix/modify it.__

I suggest you to:

1. use VM locally to run older kernel environment. (recommended, e.g. KVM/ESXi/Hyper-V)
2. try to force load a module.
3. make contact with manufacturer.

If anyone has a source code, pls share it. thx.

**Comment by izenizen** [comment]
I thank you very much for your explanation of this driver. I think the best way to install this driver on the centos6 virtual machine.
